doc/man1/openssl-pkeyutl.pod.in

```diff
@@ -79,6 +83,10 @@ then the B<-rawin> option must be also specified.
 Specifies the output filename to write to or standard output by
 default.
 
+=item B<-secret> I<filename>
+
+Specifies the output filename to write the secret to on I<-encap>.
```

Comment:
Isn't this also used (optionally) for the decapsulated output???
```suggestion
Specifies the output filename to write the I<encapsulated> secret to with I<-encap> or I<decapsulated> secret with I<-decap>.
```

Reply:
No, on `-decap` the current implementation uses only `-out` file for this purpose.